### PR TITLE
change to owner access

### PR DESCRIPTION
**Pull request by makosblade**
Unsure if this should be Owner or Write access 🤔

### CODE REVIEW

**Review by katherinebernstein — APPROVED** [review]

File: src/main/kotlin/com/openlattice/organizations/DatasetController.kt [hunk 1]
```diff
@@ -160,8 +160,8 @@ class DatasetController : DatasetApi, AuthorizingComponent {
             @PathVariable(ID) organizationId: UUID,
             @PathVariable(TABLE_NAME) tableName: String,
             @RequestBody metadataUpdate: MetadataUpdate) {
-        ensureAdminAccess()
         val tableFqnToId = getExternalDatabaseObjectFqnToIdPair(organizationId, tableName)
+        ensureOwnerAccess(AclKey(tableFqnToId.second))
```
Comment (jokedurnez) [inline src/main/kotlin/com/openlattice/organizations/DatasetController.kt:164]: should that be `ensureWriteAccess`?

**Comment by jokedurnez** [comment]
Interesting.  Is this the same as for entitysets?  Do you need owner for table-metadata changes on entity sets?

Also, what would WRITE otherwise mean on tables?

File: src/main/kotlin/com/openlattice/organizations/DatasetController.kt (hunk 1 above)
Comment (makosblade) [reply src/main/kotlin/com/openlattice/organizations/DatasetController.kt:164]: I'm not entirely sure how Write/Read/Owner map to actual postgres permissions actually but I agree that Admin is too much and that it should be either Owner or Write

**Review by geekbeast — APPROVED** [review]

**Comment by geekbeast** [comment]
Owner makes sense to me as our rule has been that you need to have owner on a permission to grant it.

**Comment by geekbeast** [comment]
Also, write would mean you could update the actual data in the table.

**Pull request merged**